js/components/record/RequiredAttributes.js

```diff
@@ -24,16 +24,6 @@ class RequiredAttributes extends React.Component {
         this.i18n = this.props.i18n;
     }
 
-    componentDidMount() {
-        const {record, formTemplate} = this.props;
-
-        if (!record.formTemplate) {
-            if (formTemplate) {
-                record.formTemplate = formTemplate;
-            }
-        }
-    }
```

Comment:
Moved to `Record.render` to make sure that record.formTemplate is set before rendering `RecordFrom`.